package.json

```diff
@@ -22,26 +24,32 @@
   },
   "homepage": "https://github.com/companieshouse/sic-code-web#readme",
   "dependencies": {
+    "@companieshouse/structured-logging-node": "^1.0.4",
     "axios": "^0.26.1",
     "express": "^4.17.3",
     "govuk-frontend": "^4.0.1",
-    "nunjucks": "^3.2.3"
+    "http-status-codes": "^2.2.0",
+    "nunjucks": "^3.2.3",
+    "uuid": "^8.3.2"
   },
   "devDependencies": {
     "@types/express": "^4.17.13",
     "@types/jest": "^27.4.1",
     "@types/node": "^17.0.21",
     "@types/nunjucks": "^3.2.1",
     "@types/supertest": "^2.0.11",
+    "@typescript-eslint/eslint-plugin": "^5.16.0",
+    "@typescript-eslint/parser": "^5.16.0",
     "copyfiles": "^2.4.1",
+    "eslint": "^8.11.0",
```

Comment:
eslint v8+ is not compatible with node v10 (the current version CH supports). Could you use `"eslint": "^7.32.0"` instead, as `7.32.0` is the latest version. that is compatible with node 10, as shown [version 7 release](https://eslint.org/docs/user-guide/migrating-to-7.0.0#-nodejs-8-is-no-longer-supported) and the [version 8 release](https://eslint.org/docs/user-guide/migrating-to-8.0.0#-nodejs-10-13-and-15-are-no-longer-supported)

Reply:
I think it's compatible as other CH services are using it: https://github.com/companieshouse/overseas-entities-web/blob/main/package.json#:~:text=%22eslint%22%3A%20%22%5E8.11.0%22%2C

I also updated the package-lock to node10 as per the comment above and the eslint v 8.11.0 still seems to run okay

Reply:
I've just cloned [that repository](https://github.com/companieshouse/overseas-entities-web) locally and running lint fails when using node v10. That service might be possibly using node v14 (which I think is nearly ready for use in CH but hasn't been rolled out across all node apps yet). Running lint on v14 works for that repository.
I think for now it may be worth continuing with node v10, which would mean using `7.32.0` for eslint.